styles.css

```diff
@@ -0,0 +1,60 @@
+
+/* ......... the header section.......... */
+header {
+  width: 375px;
+  height: 48px;
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-around;
+  color: #fff;
+  padding-top: 44px;
+}
```

Comment:
- [ ] Kindly make the header have a smaller distance between the items, you can set a `space between` and give some padding on the right-left sides of the header to look similar to the FIGMA template

![image](https://user-images.githubusercontent.com/93630700/205456415-7ff45c7b-86c5-4a08-9f96-8fd3df8818f8.png)